### ISSUE TITLE
Error if MM-list is a related content element

### ISSUE THREAD

**Issue by zonky2** [MetaModels/core#1437]
Error if MM-list is a related content element

```
[2021-03-23 16:46:54] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Could not get metamodel with id: 0" at /xyz/vendor/metamodels/core/src/ItemList.php line 554 {"exception":"[object] (RuntimeException(code: 0): Could not get metamodel with id: 0 at /xyz/vendor/metamodels/core/src/ItemList.php:554)"} []
```

* generate CE MM-List
* generate a second CE with MM-List as related content element

**Comment by zonky2** [comment]
fixed in MM 2.2

**Comment by zonky2** [comment]
Fixed with https://github.com/contao/contao/pull/2930
